doc/cabal-package.rst

```diff
@@ -1858,7 +1858,13 @@ system-dependent values for these fields.
 .. pkg-field:: default-language: identifier
    :since: 1.12
 
-   TBW
+    Specifies a group of language extensions to be activated for the project, see `here <https://downloads.haskell.org/ghc/latest/docs/users_guide/exts/control.html#controlling-extensions>` for details.
+
+    The possible values are:
+
+    -  ``Haskell2010``
+    -  ``Haskell98``
+    -  ``GHC2021``
```

Comment:
Is there a reason not to have this list “chronologically” sorted (98/2010/2021 or reversed)?

Reply:
not at all, i think newest to oldest makes the most sense?

Reply:
Fine by me!